apollo-router/Cargo.toml

```diff
@@ -68,7 +68,7 @@ askama = "0.12.1"
 access-json = "0.1.0"
 anyhow = "1.0.80"
 apollo-compiler.workspace = true
-apollo-federation = { path = "../apollo-federation", version = "=0.0.11" }
+apollo-federation = { path = "../apollo-federation", version = "=1.46.0" }
```

Comment:
Same question as above — it won't break `dev`, right?

Reply:
It won’t because `path` takes priority when `apollo-router` itself is a local path (as opposed to a crates.io dependency) 
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations